roles/common/defaults/main.yml

```diff
@@ -128,6 +128,7 @@ common:
         - libxslt1-dev
       extra_packages:
         - python-dateutil
+        - python-heatclient
```

Comment:
This list also is install at system level and not just in basevenv. Also, basevenv is missing other openstack clients (example barbican).
https://github.com/blueboxgroup/ursula/blob/f95dd3f9038f0abdf0795b385e70502ec91ea7e6/roles/common/tasks/python.yml#L105

Instead of changing it here, we should add a task to roles/clients (example: - name: install openstack clients (basevenv).
Important: when installing clients in basevenv do not use upper_constraints as that will break shade.
```